caddy.go

```diff
@@ -670,6 +671,85 @@ func Validate(cfg *Config) error {
 	return err
 }
 
+// ExportStorage calls List() and Load() on the configured
+// storage module and returns the underlying assets. 
+func ExportStorage(cfg *Config) (files []struct{
```

Comment:
I'm hoping we can find a way to put these functions -- unexported -- in the `caddycmd` package.